schematic/manifest/commands.py

```diff
@@ -121,7 +126,31 @@ def get_manifest(
         allow_none=True,
     )
 
-    def create_single_manifest(data_type):
+    def export_manifest(file_name, manifest_url, output_excel=None):
```

Comment:
do we need this function here given the one in google_api_utils?

Reply:
@milen-sage thanks for pointing this out. Where do you think this function should live?  Should it be under `create_single_manifest` or google_api_utils?

Reply:
I think google_api_utils is probably better. We can refactor later.